main_test.go

```diff
@@ -199,4 +209,59 @@ func TestDeleteProduct(t *testing.T) {
 	req, _ = http.NewRequest("GET", "/product/1", nil)
 	response = executeRequest(req)
 	checkResponseCode(t, http.StatusNotFound, response.Code)
-}
\ No newline at end of file
+}
+
+//Store test cases
+func addStore() []byte {
+	clearTable()
+	addProducts(2)
+	a.DB.Exec("INSERT INTO store(id, product_id, is_available) VALUES($1, $2, $3)", 1, 1, true)
+	a.DB.Exec("INSERT INTO store(id, product_id, is_available) VALUES($1, $2, $3)", 1, 2, false)
+
+	return []byte(`[{"name": "Product 1", "product_id": 1, "price": 10 "is_available": true},
+							{"name": "Product 2", "product_id": 2, "price": 20 "is_available": false}]`)
+}
+
+//Get store products test case
+func TestGetStoreProducts(t *testing.T) {
+	jsonStr1 := addStore()
+	req1, _ := http.NewRequest("GET", "/store/1/products", nil)
+	response1 := executeRequest(req1)
+	checkResponseCode(t, http.StatusOK, response1.Code)
+	if bytes.Compare(response1.Body.Bytes(), jsonStr1) == 0 {
+		t.Errorf("Expected list of Store; Got this -> '%v'", response1.Body.Bytes())
+	}
+}
+
+//Create Store test case
+func TestCreateStore(t *testing.T) {
```

Comment:
Can you also change this to **TestAddProductToStore**

Reply:
Renamed the function to addProductToStore - Ref: [Commit ae9967](https://github.com/nitinks-dnz/go-mux/commit/ae99670808b44316ecf3b5651678a98a284f3d00)